docs/data-retention-mechanisms.md

```diff
@@ -0,0 +1,65 @@
+# Data retention mechanisms
+
+There are several mechanisms regarding the retention of data in EmailAlertApi.
+This doc gives a brief overview of each.
+
+## Deactivating subscribers
+
+We deactivate subscribers for a couple of different scenarios. One being where
+the user unsubscribes via either following an unsubscribe link in the email
+or via the manage subscriptions page.
+
+Another way a user is deactivated is if their subscription is ended on our side
+because the topic they're signed up for no longer exists. This happens via an
+unpublished rabbitMQ message which is eventually handled by the
+[UnpublishHandleService]. This is only applicable for subscriptions to the
+topic taxonomy.
+
+[UnpublishHandleService]: https://github.com/alphagov/email-alert-api/blob/master/app/services/unpublish_handler_service.rb
+
+## Nullifying deactivated subscribers
+
+We nullify the email addresses of subscribers after they've been deactivated
+for over 28 days. This means that the address field on the subscriber record is
```

Comment:
```suggestion
for over 28 days, so that we're not storing their email address for longer than necessary.
```

I think it's worth briefly explaining why we do this. The second sentence appears to be a repeat of the first, so I've suggested we remove that too.

Reply:
Sure 👍